src/deploy-cromwell-on-azure/KubernetesManager.cs

```diff
@@ -155,7 +155,7 @@ public async Task UpdateHelmValuesAsync(IStorageAccount storageAccount, string k
             {
                 values.InternalContainersKeyVaultAuth = new List<Dictionary<string, string>>();
 
-                foreach (var container in values.DefaultContainers)
+                foreach (var container in values.DefaultContainers.Union(values.CromwellContainers))
```

Comment:
probably should ignore case, since Azure doesn't care, but we wouldn't want different case to affect this:
.Union(values.CromwellContainers, StringComparer.OrdinalIgnoreCase);